decisions/2019/05/23-d2-ui-stale-styles-etc.md

```diff
@@ -0,0 +1,42 @@
+# Front-end Decision Board, 2019-05-23
+
+## #28 How do we add cli-style to repos
+
+Considerations:
+
+- Think about if the app is suitable for feature toggling as we are
+  using it in a couple of places and getting some experience with good
+  results.
+
+- Be clear about pushing back on "backport everything, all the time",
+  especially as we move forward on continuous app deployments.
+
+**Conclusion**: We accept the proposal outlined.
+
+## #23 Use github's stale bot / autoresponder / etc.
```

Comment:
The correct issue for the stale bot / autoresponder is actually #24 and not #23

Reply:
nice, good catch.